### PR TITLE
Datasets controller specs

### PR DESCRIPTION
**Pull request by javitonino**
Some tests for the dataset controller

### CODE REVIEW

**Comment by javitonino** [comment]
CR @gfiorav

File: spec/requests/carto/builder/datasets_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+    after(:all) do
+      @visualization.destroy
+      @table.destroy
+      @map.destroy
+      CartoDB::UserModule::DBService.any_instance.stubs(:enable_remote_db_user).returns(true)
+      @user.destroy
+    end
+
+    it 'returns 404 for non-editor users requests' do
```
Comment (gfiorav) [inline spec/requests/carto/builder/datasets_controller_spec.rb]: non-builder?

**Comment by gfiorav** [comment]
Made a minor troll comment. Thanks for the specs, 📦 it!

File: spec/requests/carto/builder/datasets_controller_spec.rb (hunk 1 above)
Comment (javitonino) [inline spec/requests/carto/builder/datasets_controller_spec.rb]: Thanks! Will change. Too much copy-pasta...

**Pull request merged**